examples/CMakeLists.txt

```diff
@@ -93,6 +93,9 @@ IF(DEAL_II_COMPONENT_EXAMPLES)
             RUNTIME_OUTPUT_DIRECTORY "${CMAKE_BINARY_DIR}/${DEAL_II_EXECUTABLE_RELDIR}"
             )
           ADD_DEPENDENCIES(examples ${_name}.${_build_lowercase})
+          INSTALL(TARGETS ${_name}.${_build_lowercase}
+            DESTINATION ${DEAL_II_EXECUTABLE_RELDIR}
```

Comment:
Well, `${DEAL_II_EXECUTABLE_RELDIR}` is the canonical location for executables.

It is true that this disconnects the executable from input files. The other approach of installing the executables into a documentation directory (which `examples` ends up being) isn't particularly thrilling either. (I decided to simply not install any example binaries.)